backend/db/refresh.py

```diff
@@ -67,14 +67,16 @@ def refresh_db(
             new_data: bool = csets_and_members_enclave_to_db(con, since, schema=schema)
         except Exception as err:
             update_db_status_var('last_refresh_result', 'error', local)
-            update_db_status_var('refresh_status', 'inactive', local)
             print(f"Database refresh incomplete; exception occurred. Tallying counts and exiting.", file=sys.stderr)
             counts_update('DB refresh error.', schema, local, filter_temp_refresh_tables=True)
             counts_docs()
             raise err
+        finally:
+            refresh_complete_dt = current_datetime()
+            update_db_status_var('last_refresh_exited', refresh_complete_dt, local)  # TODO
```

Comment:
With the addition of this `last_refresh_exited`, `refresh_status` is now somewhat redundant. But we can keep it if it pleases us.